homeassistant/components/leviosa_shades/const.py

```diff
@@ -0,0 +1,35 @@
+"""Constants for the Leviosa Motor Shades Zone integration."""
+
+import asyncio
+from typing import Final
+
+from aiohttp.client_exceptions import (
+    ServerConnectionError,
+    ServerDisconnectedError,
+    ServerTimeoutError,
+)
+
+DOMAIN: Final = "leviosa_shades"
+
+MANUFACTURER: Final = "Leviosa Motor Shades LLC"
+MODEL: Final = "Zone Hub"
```

Comment:
Because it is only referenced by cover.py, it must be written in cover.py? I do not understand why? I will add service to my next PR, and I will still use the constants in const.py at that time.